src/app/shared/animations.ts

```diff
@@ -9,6 +9,11 @@ const fadeInOut = [
     transition("in => out", [animate("0.5s")]),
     // todo - could use :enter and :exit properties also (need to confirm doesn't break existing functionality)
   ]),
+  trigger("noFade", [
+    state("in", style({ opacity: 1 })),
+    state("out", style({ opacity: 0 })),
+    transition("* => in", [animate("0s")]), // this line is to allow for a prior fade to actually occur
```

Comment:
In order to wait for the previous slide's "out" animation to finish, I think potentially we want to wait 0.5s before our slide appears. 

See [these docs](https://angular.io/guide/animations#animation-metadata-duration-delay-and-easing), it may be possible to achieve by using the "delay" argument of `animate()`, along the lines of: `animate("0 0.5s")`

Reply:
Addressed: created a new setting for animation. This should be better?

https://github.com/IDEMSInternational/parenting-app-ui/assets/88288104/70cee643-16d3-4cc2-b5a3-0bf53350ad7f

Reply:
That definitely looks better in my opinion. It would be worth testing how it looks when every slide has the parameter `animation: none`. Try updating the [feat_animated_slides](https://docs.google.com/spreadsheets/d/15eQ1B7BFQKQVdAM1mD1qPa_vGrudfylYVVDOtPenkWI/edit#gid=569531329) template and see what it looks like. I suspect that with all of them not animated, the gap will look too long, you may want to reduce it from 1.5s to 0.5s, for example

Reply:
Shortened it, looks like this now:

https://github.com/IDEMSInternational/parenting-app-ui/assets/88288104/4e822854-68e7-4821-b0d3-50af744f9574